src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs

```diff
@@ -436,26 +459,63 @@ private bool CanChangeDateStepBeSkipped(ChangeStatusType changeType, GetApprenti
                         });
 
                 CheckApprenticeshipStateValidForChange(data.Apprenticeship);
+                ChangeStatusViewModel viewmodel = new ChangeStatusViewModel
+                {
+                    DateOfChange = DetermineChangeDate(changeType, data.Apprenticeship, whenToMakeChange, dateOfChange),
+                    ChangeType = changeType,
+                    WhenToMakeChange = whenToMakeChange,
+                    ChangeConfirmed = false
+                };
+
+                if (changeType == ChangeStatusType.Resume)
+                {
+                    DateTimeViewModel resumeDate = new DateTimeViewModel(data.Apprenticeship.PauseDate);
+                    DateTimeViewModel pauseDate = new DateTimeViewModel(data.Apprenticeship.PauseDate);
+                    bool academicYearBreakInTrainingHasOccured =false;
+
+                    if (!data.Apprenticeship.IsWaitingToStart(_currentDateTime))
+                    {
+                        if (data.Apprenticeship.PauseDate.HasValue)
+                        {
+                            if (data.Apprenticeship.PauseDate < _academicYearDateProvider.CurrentAcademicYearStartDate)
+                            {
+                                if (_currentDateTime.Now > _academicYearDateProvider.LastAcademicYearFundingPeriod)
+                                {
```

Comment:
We're five-deep in if/then nesting here - could we combine some of the statements or flatten this in some way?

Reply:
I will Rework the nested logic and Leverage the AcademicYearValidator at the same  time.